### PR TITLE
Add PoundForcePerFoot, PoundForcePerInch and PoundForcePerYard to ForcePerLength

### PR DESCRIPTION
**Pull request by lwestfall**

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/ForcePerLengthTests.cs [hunk 1]
```diff
@@ -31,6 +31,9 @@ public class ForcePerLengthTests : ForcePerLengthTestsBase
         protected override double MillinewtonsPerMeterInOneNewtonPerMeter => 1E3;
         protected override double NanonewtonsPerMeterInOneNewtonPerMeter => 1E9;
         protected override double NewtonsPerMeterInOneNewtonPerMeter => 1;
+        protected override double PoundsForcePerFootInOneNewtonPerMeter => 6.8521766E-2;
+        protected override double PoundsForcePerInchInOneNewtonPerMeter => 5.710147162769201E-3;
+        protected override double PoundsForcePerYardInOneNewtonPerMeter => 0.205565298;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/ForcePerLengthTests.cs]: Checked these, OK.

**Comment by angularsen** [comment]
Looks great, thanks!
Nuget 4.25.0 on the way out.

[Release UnitsNet/4.25.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.25.0)

**Comment by lwestfall** [comment]
Cool, thanks! This was also my first pull request to an open source project :)

**Comment by angularsen** [comment]
That is awesome to hear, you followed the conventions in this project perfectly!
I hope with this first PR down, you are encouraged to contibute even more, not necessarily here but in other open source projects also :-)

**Pull request merged**